stable/artifactory/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [10.41.8] - 
+* Added support for annotations on the artifactory statefulset
```

Comment:
@bmanuel  can you please add this  in just add one line
`Added support for annotations for artifactory statefulset and nginx deployment [GH-1665](https://github.com/jfrog/charts/pull/1665)`